modules/luci-mod-network/htdocs/luci-static/resources/view/network/switch.js

```diff
@@ -358,6 +358,11 @@ return L.view.extend({
 				port_opts.push(o);
 			}
 
+			o = s.option(form.Value, 'description', 'Description');
+			o.write = function(section_id, value) {
```

Comment:
Why is the `o.write()` override needed?

Reply:
Ah, ok. Didn't know it was not required. I'll remove it then